cove_ofds/process.py

```diff
@@ -231,9 +231,13 @@ def __init__(self, supplied_data):
         self.spans_file_name = os.path.join(
             self.supplied_data.data_dir(), "spans.geo.json"
         )
+        self.meta_file_name = os.path.join(
+            self.supplied_data.data_dir(), "geojson.meta.json"
+        )
 
     def process(self, process_data: dict) -> dict:
-        # TODO if original format this, don't bother
+        # TODO if original format is geojson, don't bother? or do bother?
```

Comment:
Yeah, I guess this is a question of whether we display their original data, or the round-tripped data? It feels like the latter might be useful.

Reply:
If we display the round tripped data we are essentially putting it through a layer of verification (cos bad data won't make it through). This is a general question that applies to someone uploading CSVs too, so one for later.